src/components/Todo/TodoInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todoinfo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = (props) => {
+  const { todoinfo } = props;
```

Comment:
I think it's better name this variable in camelCase:
```suggestion
  const { todoInfo } = props;
```